lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosSobolIndices.cxx

```diff
@@ -350,6 +350,50 @@ Scalar FunctionalChaosSobolIndices::getSobolGroupedIndex(const UnsignedInteger v
   return getSobolGroupedIndex(index, marginalIndex);
 }
 
+/* Sobol grouped total index accessor */
+Scalar FunctionalChaosSobolIndices::getSobolGroupedTotalIndex(const Indices & variableIndices,
+    const UnsignedInteger marginalIndex) const
+{
+  // Compute the complementary indice
+  const UnsignedInteger inputDimension = functionalChaosResult_.getDistribution().getDimension();
+  const UnsignedInteger groupSize = variableIndices.getSize();
+  const UnsignedInteger complementaryGroupSize = inputDimension - groupSize;
```

Comment:
you need to check that the size of variableIndices doesnt exceed dimension else this will overflow

Reply:
You are right: this is fixed. 
As far as I understand, the current policy is to check the input arguments at the level where it is appropriate: am i correct ?

Reply:
yes